content/applications/inventory_and_mrp/purchase/advanced/vendor_costs_report.rst

```diff
@@ -0,0 +1,119 @@
+   under the :guilabel:`Filters` column.
+
+Under the :guilabel:`Filters` column, select a date range to use for comparison. The report can be
+filtered by either :guilabel:`Order Date` or :guilabel:`Confirmation Date`. Choose one from the
+list, and click the |caret| to specify the date range, either by month, quarter, or year.
+
+Next, under the :guilabel:`Group by` column, select :guilabel:`Vendor`. Then, select
+:guilabel:`Product`, which is also located in the :guilabel:`Group By` column.
+
+.. note::
+    Selecting :guilabel:`Product` is **not** required for this report. However, it is recommended,
+    as it provides additional insight into the performance of individual vendors. Additional
+    selections can be made under the :guilabel:`Group by` heading as well, including
+    :guilabel:`Product Category`, :guilabel:`Status`, and :guilabel:`Purchase Representative`.
+
+    To ensure the report is generated correctly, make sure that :guilabel:`Vendor` is the **first**
+    selection made under the :guilabel:`Group By` column.
+
+Next, make a selection under the :guilabel:`Comparison` heading. These options are only available
+after the date range is selected under the :guilabel:`Filters` column, and vary based on that range.
+:guilabel:`Previous Period` adds a comparison to the previous period, such as the last month or
+quarter. :guilabel:`Previous Year` compares the same time period from the previous year.
+
+.. note::
+   While multiple time-based filters can be added at once, only one comparison can be selected at a
+   time.
+
+.. image:: vendor_costs_report/filters-groups.png
+   :align: center
+   :alt: The drop-down menu of filters, group by and comparison options for the vendor costs report.
+
+Add measures
+------------
+
+After selecting the :guilabel:`Filters`, :guilabel:`Group by`, and :guilabel:`Comparison` settings,
+click out of the drop-down menu.
+
+By default, the report displays with the following measures: :guilabel:`Order`, :guilabel:`Total`,
+:guilabel:`Untaxed Total`, and :guilabel:`Count`. Click :guilabel:`Measures` at the top-left to open
+the drop-down list of available measures. Click :guilabel:`Average Cost` to add it to the report.
+Select any additional measures to add to the report, or click on any of the already selected
+measures to remove them, if desired.
+
+.. tip::
+   It is recommended to run the report with at least :guilabel:`Average Cost`, :guilabel:`Total`, or
+   :guilabel:`Untaxed Total` selected from the :guilabel:`Measures` list. Additional measures, such
+   as :guilabel:`Days to Receive`, can be added to provide additional insights.
+
+View results
+============
+
+The vendor costs report is displayed in the *pivot* view, by default. Click :guilabel:`Insert in
+Spreadsheet` to add the pivot view into an editable spreadsheet format within the *Dashboards* app.
+If the Odoo *Documents* app is installed, the report can be inserted into a blank or existing
+spreadsheet, and exported.
```

Comment:
Arent spreadsheets editible from within the *Documents* app? And the Insert into Spreadsheet button will only appear if the specific _Documents Spreadsheets_ module is installed, which installs the *Spreadsheets* module as a dependency 
```suggestion
Spreadsheet` to add the pivot view into an editable spreadsheet format within the *Documents* app.

.. important::
   The :guilabel:`Insert in Spreadsheet` option is only available if the *Documents Spreadsheet*
   module is installed.
```